torch/_meta_registrations.py

```diff
@@ -722,6 +727,13 @@ def linalg_lu_solve_meta(
         if result.is_complex():
             result = result.conj()
 
+    if out is not None:
+        assert isinstance(out, TensorLike)
+        out = _maybe_resize_out(out, result.shape)
+        out = _safe_copy_out(copy_from=result, copy_to=out)  # type: ignore[arg-type]
+        # sets conj bit in out (not done by out_wrapper)
+        torch._C._set_conj(out, result.is_conj())
+        return out
```

Comment:
This is `out_wrapper` inlined with `_set_conj` to match the code in core (pasted in the other comment).

There might be problems with setting the conj bit on `out` when aliasing is used:
- https://github.com/pytorch/pytorch/pull/101835#pullrequestreview-1435573272
- https://github.com/pytorch/pytorch/pull/101835#issuecomment-1556254834

Unlike in the fft ops, there's no way to return from `TORCH_IMPL_FUNC(linalg_lu_solve_out)`. You can only write to `result`. So you cannot use `conj_physical ` for out. Is this the best way we can do here?

Reply:
cc @ezyang (see above)

Reply:
IMO, it's alright if we don't model fully well the conj bit in these ops. There's just two ops that perform optimisations via the conj bit to avoid a full copy, and that's SVD (in some cases) and this op. I would say we simply do `result = torch.con(result)`  here and move on. This would allow us to use `out_wrapper`.

Reply:
Nope, it won't work the way you describe with `out_wrapper`, unfortunately. I've already tried it. The issue is that copy in out_wrapper doesn't copy the conj bit. So the only way is to keep it as is AFAICT.

Reply:
>IMO, it's alright if we don't model fully well the conj bit in these ops.

conj bit is an implementation detail, and it's alright if it's not modelled perfectly well in `svd` and `linalg_lu_solve.

Reply:
OK, can I just skip the conj test that fails, then?

Reply:
I think so, unless @ezyang feels strongly against it.

Reply:
Changed it back to use out_wrapper. Had to add a new conditional to skip conj test failures.

Reply:
So what do you want me to look at here? I agree the conj code looks kind of funky, but if it agrees with tests, it may just be a preexisting problem.